var/spack/repos/builtin/packages/py-fair-esm/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyFairEsm(PythonPackage):
+    """Evolutionary Scale Modeling"""
+
+    homepage = "https://github.com/facebookresearch/esm"
+    pypi = "fair-esm/fair-esm-2.0.0.tar.gz"
+
+    license("MIT")
+
+    version("2.0.0", sha256="4ed34d4598ec75ed6550a4e581d023bf8d4a8375317ecba6269bb68135f80c85")
```

Comment:
Looking at the setup.py, `esmfold` should be a variant with the dependencies listed under that

```
variant("esmfold", default=True, description="Enable dependencies for OpenFold")
depends_on("py-biopython", when="+esmfold", type=("build", "run"))
...
# other deps here
```

Reply:
done